packages/compass/package.json

```diff
@@ -150,6 +150,8 @@
     "email": "compass@mongodb.com"
   },
   "dependencies": {
+    "@emotion/cache": "^11.7.1",
+    "@emotion/serialize": "^1.0.2",
```

Comment:
Do we need to have these dependencies here? How do we make sure that they are aligned with leafygreen? Do you think we can use `injectGlobal` export from leafygreen, inject just one style, and use class names / data-attr selectors on `:root` to maintain themed styles (which is a very common approach for global styles in my experience) instead of doing all this emotion cache management?

Reply:
Yeah I think it would be good to get rid of these deps, thanks for bringing it up. Will look into alternatives like your suggestion and ping back when updated.

Reply:
@gribnoysup Updated to use `injectGlobal` to add the styles once, and then theming using `data` selectors. Added tests too.

Reply:
Nice, seems like a lighter implementation to me 👍